### PR TITLE
Create better status bar API + Structure, add time to status bar

### PR DESCRIPTION
**Pull request by joshua-s**

### CODE REVIEW

File: laskyawm-clock.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	var clock = document.createElement('span');
+	clock.classList.add('barButton');
+	clock.id = "bar-clock";
+	updateTime();
+	
+	// Add to bar
+	bar.addItem(clock);
+
+	// Update clock on time changes (500 to account for errant synchronization)
+	setInterval(updateTime,500);
```
Comment (twiss) [inline laskyawm-clock.js]: Could you line this up entirely by doing a `setTimeout` with `1000 - currentTime.getMilliseconds()` after you update the time?

File: laskyawm-bar.js [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+	document.body.appendChild(barElement);
+
+	// Add icon panel
+	var barIcons = document.createElement('div');
+	barIcons.id = 'bar-icons';
+	barElement.appendChild(barIcons);
+	
+	bar.addItem = function (node) {
+		$(barIcons).prepend(node);		
+	}
```
Comment (twiss) [inline laskyawm-bar.js]: Could you run this through the style checker? Instructions here: https://github.com/airbornos/airbornos/blob/4388129/CONTRIBUTING.md, and then `git commit --amend`

**Comment by twiss** [comment]
Cool! Could you horizontally center the clock and have the cursor be default? Also, was it intentional to make the power button larger?

**Comment by joshua-s** [comment]
If we are using the status bar for minimized (or maximized #6) windows, does it make sense to center the clock? I have addressed all your other comments!

**Comment by twiss** [comment]
Er, I meant vertically center. Sorry.

**Comment by joshua-s** [comment]
Done!

**Comment by twiss** [comment]
:+1:. Two more things: the `barMenu`s look a little weird now, could you give `#togglePowerMenu` and `.barMenu` the old padding? And two, it doesn't fit on mobile, you could remove the clock there. There's a `deviceType` variable.

**Comment by twiss** [comment]
Merged as cec9cb4. Thanks!

**Pull request closed without merging**